config/config-example.js

```diff
@@ -413,18 +413,24 @@ exports.disablehotpatchall = false;
  *     - hotpatch: /hotpatch, /crashfixed and /savelearnsets commands.
  *     - ignorelimits: Ignore limits such as chat message length.
  *     - promote: Promoting and demoting. Will only work if the target user's current
- *                  group and target group are both in jurisdiction.
+ *                  group and target group are both in jurisdiction. Setting this to
+ *                  true will fallback to the previous jurisdiction's permission level.
  *     - room<rank>: /roompromote to <rank> (eg. roomvoice)
  *     - makeroom: Create/delete chatrooms, and set modjoin/roomdesc/privacy
  *     - editroom: Editing properties of rooms
  *     - editprivacy: Set modjoin/privacy only for battles
- *     - ban: Banning and unbanning.
- *     - mute: Muting and unmuting.
- *     - lock: locking (ipmute) and unlocking.
+ *     - ban: Banning and unbanning. Setting to true will fallback to the
```

Comment:
I wouldn't phrase it this way. It doesn't fall back to the previous jurisdiction, it falls back to the current group's default jurisdiction.

I'd also just phrase all of the individual permissions as "Target subject to jurisdiction", and describe how jurisdiction works at the top of the comment.

Reply:
Something like this?
```
 *     - promote: Promoting and demoting. Will only work if the target user's current
 *                  group and target group are both in jurisdiction. Target is subject to
 *                  jurisdiction. Setting this to true will fallback to the current group's
 *                  default jurisdiction.
```

Reply:
More like

```
 *     - promote: Promoting and demoting. Will only work if the target user's current
 *                  group and target group are both in jurisdiction. Has a target.
```